packages/psss/src/api/queries/useWeeklyReportForAlert.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import groupBy from 'lodash.groupby';
+import keyBy from 'lodash.keyby';
+import { reduceToDictionary } from '@tupaia/utils';
+import { getPeriodByDate } from '../../utils';
+import { combineQueries, useData, useReport } from './helpers';
+
+export const useWeeklyReportForAlert = alert => {
+  const { period, organisationUnit: countryCode, syndrome } = alert;
+
+  const reportParams = {
+    startWeek: period,
+    endWeek: getPeriodByDate(new Date()),
+    syndrome,
+  };
+  const query = combineQueries({
+    countryReport: useReport(`confirmedWeeklyReport/${countryCode}`, {
+      params: reportParams,
+    }),
+    siteReport: useReport(`weeklyReport/${countryCode}/sites`, {
+      params: reportParams,
+    }),
+    sites: useData(`country/${countryCode}/sites`),
+  });
+
+  const { countryReport, siteReport, sites = [] } = query.data;
+  const countryDataByWeek = keyBy(countryReport, 'period');
+  const siteDataByWeek = groupBy(siteReport, 'period');
+  const orgUnitCodeToName = reduceToDictionary(sites, 'code', 'name');
+
+  const data = Object.values(countryDataByWeek).map(countryData => {
```

Comment:
My understanding of this is that you could just iterate on `countryReport` instead of `countryDataByWeek` right?

Reply:
Oops yes good call! I guess this is a remnant from a previous version of the algorithm, will change it